intermediate_source/tensorboard_profiler_tutorial.py

```diff
@@ -0,0 +1,242 @@
+"""
+PyTorch Profiler With TensorBoard
+====================================
+This recipe demonstrates how to use PyTorch Profiler
+to detect performance bottlenecks of the model.
```

Comment:
let's say that this tutorial demonstrates how to use TensorBoard plugin with PyTorch Profiler to detect performance bottlenecks of the model